examples/.pytest_cache/v/cache/nodeids

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I don't think we should be tracking these - .pytest_cache should be in our .gitignore